worker/src/router.rs

```diff
@@ -332,17 +340,15 @@ impl<'a, D: 'static> Router<'a, D> {
             }
         }
 
-        if let Some(handler) = not_found_handler {
+        if let Ok(Match { value, .. }) = or_else_any_method_handler.at(&req.path()) {
             let route_info = RouteContext {
                 data,
                 env,
                 params: HashMap::new(),
```

Comment:
I think we'd still want to collect the params from the `Match` into the `RouteContext` so that a user can easily access them from the `ctx.param` func - right? 

I could see a scenario where someone might want to handle the `or_else_any_method` case differently based on the route params. 

Curious to know if you had other thoughts and specific reasoning to not pass them through and create an empty `HashMap` instead?

Reply:
You're absolutely right, that was just an oversight on my part. I wanted everything to compile with minimal changes first and then completely forgot to pass the params!

I fixed it on my branch in case you want to take the commit from there, but of course feel free to just do it here otherwise.

Reply:
Ok cool - yea, I sort of figured as much :) 

I have a `From` impl on `matchit::Params` for the `RouteContext` which I also changed to be a newtype over the HashMap .. so I'll probably just carry on with the changes I have, but thank you for updating so quickly!